### DISCUSSION TITLE
Sunbird Saral installation locally

### DISCUSSION THREAD

**Discussion by Kovendan03** [Sunbird-Saral/Community#26] [General]
Hi Team,

I have installed the Sunbird Saral locally in windows using the following documentation - https://saral.sunbird.org/use/workspace-setup-playbook, able to see the app in AVD. when i tried to login, it asks me for a school id and password. Do we need to generate a id  for this as it is part of data import?

**Comment by dileep-gadiraju** [comment]
Hi @Kovendan03,

If you are running a reference backend with NodeJS and MongoDB, you can import reference data using the commands given below. 
https://saral.sunbird.org/use/workspace-setup-playbook#backend-data-load

Also, refer to the below JSON for v1.5.5 release:

https://github.com/Sunbird-Saral/Project-Saral/blob/v1.5.5/v1.0/backend/data/schools.json
hope this helps.
